docs/source/reference/command_line.rst

```diff
@@ -313,6 +313,43 @@ Below is a list with all available subcommands.
       list  Display a list of all available plugins.
 
 
+.. _reference:command-line:verdi-presto:
+
+``verdi presto``
+----------------
+
+.. code:: console
+
+    Usage:  [OPTIONS]
+
+      Set up a new profile as automatically as possible.
+
+      This command aims making setting up a new profile as easy as possible. It intentionally
+      provides a limited amount of options to customize the profile and by default does not
+      require any options to be specified at all. For full control, please use the command
+      `verdi profile setup` instead.
+
+      The main goal for this command is to setup a profile with minimal requirements to make
+      it easy to install AiiDA and get started as quickly as possible. Therefore, by default,
+      the created profile uses the `core.sqlite_dos` storage plugin which does not require any
+      services, such as PostgreSQL and RabbitMQ are not required. This _does_ mean, however,
+      that not all functionality of AiiDA is available, most notably running the daemon and
+      submitting processes to said daemon.
+
+      The command performs the following actions:
+
+      * Create a new profile that is set as the new default
+      * Create a default user for the profile (email can be configured through options)
+      * Set up the localhost as a `Computer` and configure it
+      * Set a number of configuration options with sensible defaults
```

Comment:
I wonder if there is a small set of most impactful configuration options that should be mentioned? Or at least we could link to some section where more info can be found (maybe the `verdi profile setup` command)?

Reply:
You mean just list here the options that are set? And why the `verdi profile setup` command? That doesn't allow directly to set configuration options.

Reply:
Yeah, just mention the options that are set. But feel free to ignore this.

Tbh I am still really confused about the difference between profile setup and configuration. Why is there a split like that? (Convo for another time, please ignore)